### ISSUE TITLE
Dashboard

### ISSUE THREAD

**Issue by bheiskell** [bheiskell/Riiga#17]
- Have a link to specific information pages including rules and wiki. This is dependent on issue #117.
- Display last five entries made in stories belonging to the user (one entry per story)
- List all the user's stories, but sort so that the ones waiting on the current user are listed at the top
- List the stories that have less than one active member but are not marked complete
- Bookmarked threads (needs a new issue)

**Comment by Treijim** [comment]
Possibly another area where it shows threads which they aren't involved in, but which they've "bookmarked" due to interest?
